.gitignore

```diff
@@ -2,3 +2,4 @@
 /iroh_gateway/test_files/*
 .env
 Cargo.lock
+.idea
```

Comment:
please remove, you can always use a personal gitignore for this: https://tekin.co.uk/2020/03/maintain-a-global-git-ignore-file

Reply:
Done